### PR TITLE
Adds Booze/Soda Dispenser boards

### PR DESCRIPTION
**Pull request by ChilbroBaggins**
Changelog:
-Adds machine circuit boards for the booze and soda dispensers.

I didn't notice any other missing ones but if there are some, let me know, I can add to this PR. 

(Image/CL not necessary)

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml [hunk 1]
```diff
@@ -677,3 +677,33 @@
...
+  description: A machine printed circuit board for a booze dispenser
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: booze_dispenser
+      requirements:
+        Capacitor: 1
+        Manipulator: 1
+        MatterBin: 1
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml:694]: just make these use steel + beakers if you aren't going to implement machine upgrading.

File: Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml [hunk 2]
```diff
@@ -677,3 +677,33 @@
...
+  description: A machine printed circuit board for a soda dispenser
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: soda_dispenser
+      requirements:
+        Capacitor: 1
+        Manipulator: 1
+        MatterBin: 1
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml:709]: same for this one

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
you added the circuitboards but there's no way of actually acquiring them. consider a "food services" tech with these + maybe microwave.

**Comment by ChilbroBaggins** [comment]
> you added the circuitboards but there's no way of actually acquiring them. consider a "food services" tech with these + maybe microwave.

I could move the knives and Micro out of "Biological tech" into this new category. That would leave only the Scythe, hatchet and shovel recipes. Should I maybe add some recipes for the other Hydro tools, clippers, spade...is that all of them?

**Comment by EmoGarbage404** [comment]
in the future those tools could have recipes, but for this pr, feel free to just only move the microwave.

**Comment by ChilbroBaggins** [comment]
Looks like the recipes already exist. Just not added to the tech tree.

File: Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml [hunk 3]
```diff
@@ -677,3 +677,33 @@
...
+- type: entity
+  id: BoozeDispenserMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: booze dispenser machine board
+  description: A machine printed circuit board for a booze dispenser
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: booze_dispenser
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml:690]:
```suggestion
      prototype: BoozeDispenser
```

Comment (ChilbroBaggins) [reply Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml:690]: I saw that. It's inconsistent in the file. But I noticed the majority use this format. So I changed it.

**Comment by ChilbroBaggins** [comment]
I'm sorry this is becoming sloppy. I have a weird headfog thing going on today. I need to head to work soon. I'll finish later if I don't get this sorted before that. I know I need to find the string for the tech tree yet but I'm not seeing it in the Research folder.

**Comment by EmoGarbage404** [comment]
> I'm sorry this is becoming sloppy. I have a weird headfog thing going on today. I need to head to work soon. I'll finish later if I don't get this sorted before that. I know I need to find the string for the tech tree yet but I'm not seeing it in the Research folder.

Its under prototypes iirc.

**Comment by ChilbroBaggins** [comment]
went ahead an updated the botany research/recipe stuff [#13252](https://github.com/space-wizards/space-station-14/pull/13252)

File: Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl [hunk 4]
```diff
@@ -4,6 +4,9 @@ technologies-basic-research-technology-description = Nanotrasen basic research t
 technologies-cleaning-technology = Cleaning technology
 technologies-cleaning-technology-description = Start to a shiny clean station.
 
+technologies-FoodBev-technology = Food and beverage technology
```
Comment (Peptide90) [inline Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl:7]: Looks good from a cursory look however FoodBev doesn't fit the nomenclature of the other localisation strings.

File: Resources/Prototypes/Catalog/Research/technologies.yml [hunk 5]
```diff
@@ -30,6 +30,25 @@
   - SprayBottle
   - BodyBag
 
+# Food/Bev Service Technology Tree
+
+- type: technology
+  name: technologies-FoodBev-technology
```
Comment (Peptide90) [inline Resources/Prototypes/Catalog/Research/technologies.yml:36]: And here

**Review by Peptide90 — APPROVED** [review]
Looking good, only comments is what I picked up on but very minor.

File: Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl (hunk 4 above)
Comment (ChilbroBaggins) [reply Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl:7]: Is it the upper case or abbreviation I should change? or both?

Comment (ChilbroBaggins) [reply Resources/Locale/en-US/prototypes/catalog/research/technologies.ftl]: Well, I lower-cased it. Hopefully, that's what you meant.

**Review by EmoGarbage404 — APPROVED** [review]

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**